src/main.js

```diff
@@ -1,13 +1,63 @@
+const fs = require('node:fs/promises')
+const xml2js = require('xml2js')
 const core = require('@actions/core')
 
+let reportContent = `
+# Mutation Test Summary
+        
+## Overview
+
+This report provides an overview of mutation testing results, grouped by file. Each entry details a mutation attempt, its detection status, and specific mutation description.
+
+`.trimStart() // Removes the first new line, before "# Mutation Test Summary"
+
 async function run() {
+  // Read inputs of the action
   const filePath = core.getInput('file-path', { required: true })
   const displayOnlySurvived =
     core.getInput('display-only-survived', { required: false }) === 'true'
```

Comment:
FYI, `core.getInput` could still return `undefined`. Probably you'd like to check the WF inputs are valid before continuing or fail fast otherwise

Reply:
BTW, for `displayOnlySurvived` you could use `core.getBooleanInput`
See: https://github.com/actions/toolkit/tree/main/packages/core#inputsoutputs

Reply:
Aha, my assumption was that the action would fail from the CI step if the required field from the manifest is not there - good catch!

As for the boolean, it's a good idea! 🙏

Reply:
Looking at the documentation now,

```js
    /** Optional. Whether the input is required. If required and not present, will throw. Defaults to false */
    required?: boolean;
``

So to the best of my understanding, we don't actually need to validate it's truly set

Reply:
interestingly enough intellij complains about invalid schema - but it seems to work :)
<img width="555" alt="image" src="https://github.com/yonatankarp/pitest-summary/assets/14914865/18116281-958f-4fe0-8ec8-b6285771c063">